tutorial/conan_repositories/setting_up_conan_remotes/artifactory/artifactory_ce_cpp.rst

```diff
@@ -17,20 +17,56 @@ Running Artifactory CE
 
 There are several ways to run Artifactory CE:
 
-* **Running from a Docker image:**
+Running from a Docker image
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+As of Artifactory 7.77, it is mandatory to use an additional SQL database such as
```

Comment:
"it is mandatory to use an additional SQL database" -> it is a requirement to point Artifactory to a running instance of a SQL server"  (such as...)